component/admin/models/webservice.php

```diff
@@ -73,6 +73,29 @@ class WebservicesModelWebservice extends JModelAdmin
 	 */
 	public function __construct($config = array())
 	{
+		if (empty($config['filter_fields']))
+		{
+			$config['filter_fields'] = array(
+				'published', 'w.published',
+				'client', 'w.client',
+				'state', 'w.state',
+				'path', 'w.path',
+			);
+		}
+
+		try
+		{
+			$container = (new Joomla\DI\Container)
+				->registerServiceProvider(new Joomla\Webservices\Service\ConfigurationProvider)
+				->registerServiceProvider(new Joomla\Webservices\Service\DatabaseProvider);
+		}
+		catch (\Exception $e)
+		{
+			throw new RuntimeException('Help!', 500);
```

Comment:
Oops I forgot about these dummy exceptions. Fixed now (both nesting and comments)